README.md

```diff
@@ -1,7 +1,25 @@
+![W3C Logo](https://www.w3.org/Icons/w3c_home)
+
 # Publishing Community Group
 
 This is the repository for ongoing work done by the [W3C Publishing Community Group](https://www.w3.org/community/publishingcg/).
 
+## Specifications
+
+T.B.D
+
+## COntributing to the Publishing Community Group Repositories
+
+Use the standard fork, branch, and pull request workflow to propose changes to the specification. Please make branch names informative窶巴y including the issue or bug number for example.
```

Comment:
Jeff, something went wrong and you mixed in some Chinese characters... I presume you wanted to say 'informatively' :-)

Reply:
Lol, yes will fix. Thanks Ivan

Reply:
done